repositories/dbmodels/db_phantom_decision.go

```diff
@@ -0,0 +1,21 @@
+package dbmodels
+
+import "time"
+
+const TABLE_PHANTOM_DECISIONS = "phantom_decisions"
+
+type DbPhantomDecision struct {
+	Id                  string    `db:"id"`
+	OrganizationId      string    `db:"org_id"`
+	CreatedAt           time.Time `db:"created_at"`
+	Outcome             string    `db:"outcome"`
+	ScenarioId          string    `db:"scenario_id"`
+	ScenarioVersion     int       `db:"scenario_version"`
+	Score               int       `db:"score"`
+	TriggerObjectRaw    []byte    `db:"trigger_object"`
```

Comment:
thinking about this again, I think we can actually remove (because not necessary):
TriggerObjectRaw
TriggerObjectType
PivotId
PivotValue
ScenarioVersion


None of those will be useful for computing the analytics which is the only way we'll consume the phantom decisions